tests/python/pants_test/pantsd/pantsd_integration_test_base.py

```diff
@@ -37,7 +37,7 @@ def _log(self):
       'PantsDaemonMonitor: pid is {} is_alive={}'.format(self._pid, self.is_alive()))
     )
 
-  def assert_started(self, timeout=.1):
+  def assert_started(self, timeout=12):
```

Comment:
This is a pretty drastic bump... What causes the increase?

Reply:
Most of the pantsd integration tests would fail from a timeout while awaiting the pid until I increased it to 6 seconds locally, and I would still see sporadic CI failures until I decided to double that. It's not clear to me why it's necessary to wait this long here.

Reply:
Was this the case before your changes as well as after?

Reply:
I'd also be curious to know why this didn't show on CI before, but I can check that.
In either case, it might be worth adding a comment here about why that specific number?

Reply:
> Was this the case before your changes as well as after?

Yes, I have been unable to run pantsd integration tests locally for I believe ever? The CI part may be related to the long sleep I've added and may be able to remove.